feature/SM-4128: Inspection Units and Final Reinstatements (#111)
departmentfortransport/street-manager-docs: docs/api-documentation/V1.10/index.md

If a permit has been stopped accidentally, it can be reverted using this endpoint. This will remove the actual end date and change the active permit's status back to in-progress.
{: .govuk-body}

#### Inspection units endpoint
{: .govuk-heading-s}

<code>PUT /works/{work reference number}/inspection-units</code>

> Once a permit is in progress and an excavation has been carried out, a promoter can log the inspection units to the Works Record.

Reviewer: ```suggestion
Once a permit is in progress, and an excavation has been carried out, a promoter can log the inspection units to the Works Record.
```

{: .govuk-body}

> You can only provide inspection-units if a reinstatement currently exists on the works record, which will cover in a separate section.

Reviewer: ```suggestion
You can only provide inspection units if a reinstatement currently exists on the works record, which will cover in a separate section.
```

Reviewer: Also what do you mean by "which will cover in a separate section"?

> You can only provide inspection-units if a reinstatement currently exists on the works record, which will cover in a separate section.

Reviewer 2: ```suggestion
You can only provide inspection-units if a reinstatement currently exists on the works record, which is covered in a separate section.
```?

{: .govuk-body}

#### Final reinstatement endpoint
{: .govuk-heading-s}

<code>PUT /works/{work reference number}/final-reinstatement</code>

Once a permit is in progress, and an excavation site has been added to the work, a promoter can flag that the final site has been registered for that work.
{: .govuk-body}

> You can only provide final-reinstatement if an excavation reinstatement currently exists on the works record, which is covered in a separate section.

Reviewer: ```suggestion
You can only provide final reinstatement if an excavation reinstatement currently exists on the works record, which is covered in a separate section.
```
